docs/source/tutorial.rst

```diff
@@ -529,7 +529,7 @@ files <External-variables-with-cfa>`.
 **Visualisation**
 -----------------
 
-Powerful, flexible, and very simple to produce visualisations of field
+Powerful and flexible visualisations of field
```

Comment:
Could we somehow retain the notion that cf-plot is very easy to use (e.g. not all plotting libraries allow you to produce a plot with a one-liner) (same as earlier)